src/routes/Edit.svelte

```diff
@@ -136,6 +172,25 @@
         };
         updateCodeStore(newState);
     }
+
+    function loadHistory(history) {
+        let historyList = document.getElementById("historyList");
+        if (!historyList) return;
+        historyList.innerHTML = "";
+        history.forEach(element => {
+            let button = document.createElement("button");
+            button.className = "button-style";
+            button.innerHTML = element.time;
+            button.onclick = function () {
+                updateCodeStore({
+                    code: element.code,
+                    mermaid: { theme: "default" },
+                    updateEditor: true,
+                });
+            };
+            historyList.appendChild(button);
+        });
+    }
 </script>
```

Comment:
Wouldn't it be better to leave the DOM modification for svelte to handle (using templates and loops) and simply update the historyList array here?

Reply:
Thanks for the suggestion, it has been optimized

Reply:
Awesome! 💯